examples/angular/simple/src/simple-example.component.ts

```diff
@@ -0,0 +1,43 @@
+import { AngularQueryDevtoolsComponent } from '@tanstack/angular-query-devtools-experimental'
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { CreateQuery } from '@tanstack/angular-query-experimental'
+import axios from 'axios'
+
+type Response = {
+  name: string
+  description: string
+  subscribers_count: number
+  stargazers_count: number
+  forks_count: number
+}
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'simple-example',
+  standalone: true,
+  template: `
+    <ng-container *ngIf="query().isPending">Loading...</ng-container>
```

Comment:
I'd like to see the query being a normal object and `isPending` being a signal, so we can be surgical in template updates (when signal components get released)

Same thing for error and data. 

This way only the part of the template that has a signal update will get updated.

Reply:
I put this on my todo list to look at. Not sure how much work it would be.